app/pages/_app.js

```diff
@@ -7,11 +7,10 @@ import { initAuth } from '../components/auth/firebase';
 initAuth();
 
 function MyApp({ Component, pageProps }) {
-  return (
+  const getLayout = Component.getLayout || ((page) => <Layout menu={pageProps}>{page}</Layout>)
+  return getLayout(
```

Comment:
Are we sure this change doesn't make all pages not SSR and/or static gen-able?

Reply:
Yeah, actually the navbar doesn't go well with the theme so I thought of going with the design completely. Any suggestions of what should we do here?